qa/common/gen_qa_model_repository

```diff
@@ -55,9 +55,9 @@ ONNX_VERSION=1.16.1
 ONNX_OPSET=0
 
 # OPENVINO version
-OPENVINO_VERSION=2024.5.0
+OPENVINO_VERSION=2024.3.0
```

Comment:
This version should be:
```suggestion
OPENVINO_VERSION=2024.4.0
```
according to: https://github.com/triton-inference-server/server/pull/7811